lib/event_people/rabbit_broker.go

```
@@ -92,14 +92,30 @@ func (rabbit *RabbitBroker) Consume(eventName string, callback Callback) {
 	if err != nil {
 		log.Println(err)
 	}
+	queueName := queue.queueNameByRoutingKey(eventName)
+	publish := func(routingKey string, headers amqp.Table, body []byte) error {
+		return channel.Publish("", routingKey, false, false, amqp.Publishing{
```

golangci-lint (2.12.2)

[error] SA1019: channel.Publish is deprecated: Use PublishWithContext instead. 

(staticcheck)